src/goose.rs

```diff
@@ -913,6 +910,153 @@ impl GooseUser {
         Ok(single_user)
     }
 
+    /// Returns a reference to GooseUser's session data.
+    /// Leaves the session in-place, creating a new one Option with a reference
+    /// to the original session data.
+    ///
+    /// Return none if no session data has been set or that the session data is not of type `T`
+    ///
+    /// # Example
+    /// ```rust
+    /// use goose::prelude::*;
+    ///
+    /// #[derive(Debug, Clone)]
```

Comment:
Thanks to your PR, this `Clone` isn't required, let's remove from the documentation example.

Reply:
Oh yes, my bad.